include/boost/math/tools/engel_expansion.hpp

```diff
@@ -56,15 +57,15 @@ class engel_expansion {
             computed += term;
         }
 
-        for (size_t i = 1; i < a_.size(); ++i)
+        for (size_t j = 1; j < a_.size(); ++j)
         {
             // Sanity check: This should only happen when wraparound occurs:
-            if (a_[i] < a_[i-1])
+            if (a_[j] < a_[j-1])
```

Comment:
Did I accidently alias `i` in a nested loop?

Reply:
The change suppressed shadowing warnings because `i` is [defined here](https://github.com/boostorg/math/blob/63a16971afc470e1772ed99da2ed2a2835a8954c/include/boost/math/tools/engel_expansion.hpp#L41).